include/kernel.h

```diff
@@ -3610,7 +3630,7 @@ void k_msgq_cleanup(struct k_msgq *q);
  * @retval -EAGAIN Waiting period timed out.
  * @req K-MSGQ-002
  */
-__syscall int k_msgq_put(struct k_msgq *q, void *data, s32_t timeout);
+__syscall int k_msgq_put(struct k_msgq *msgq, void *data, s32_t timeout);
```

Comment:
similar comment about arg renaming, just makes this a bit more difficult to review.